### PR TITLE
Something more lightweight...

### PR DESCRIPTION
**Pull request by rsa-ingo**
Except the darkmode switcher, all UI is now loaded via CDNs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,8 @@ Such betrayals remain unknown to the network until the network actively and at h
 Start Playing Now
 -----------------
 
-Click [here](https://ethanheilman.github.com/flipIt/playable_with_instructions.html) to play flipIt against the computer.
+Click [here](https://rsa-ingo.github.io/flipIt/flipit.html) to play flipIt against the computer or let the computer play against itself.
```
Comment (EthanHeilman) [inline README.md:18]:
```suggestion
Click [here](https://ethanheilman.github.io/flipIt/flipit.html) to play flipIt against the computer or let the computer play against itself.
```

File: js/flipit.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
  * @author Ethan Heilman 
- *
+ * @author Ingo Schubert
```
Comment (EthanHeilman) [inline js/flipit.js:3]: <3

**Review by EthanHeilman — APPROVED** [review]

**Comment by EthanHeilman** [comment]
I really appreciate you giving back to this repo. This is an excellent improvement

Do you object to your code in this PR being licensed under APACHE2.0? I plan to add a license to the repo soon

**Comment by rsa-ingo** [comment]
No objections at all. I'm fine with Apache 2.0 license.

**Pull request merged**